### DISCUSSION TITLE
Help: How to use Q&A mode when creating a knowledge base, I use version 0.6.16

### DISCUSSION THREAD

**Discussion by tarrett** [langgenius/dify#7016] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I cannot find the Q&A mode in the Web page when create a new knowledge base, the dify version is 0.6.16

**Comment by phoben** [comment]
I also encountered the same problem
